### PR TITLE
Remove Micro Site/Water Balance Site

### PR DESCRIPTION
**Pull request by arottersman**
There's nothing comparable to the MMW Micro Site in the bee project, so this PR removes all reference to it.

### Testing
- Pull this branch, restart the server
- Confirm you can still navigate the app
- Go to `localhost:8000/micro` --> you should get a 404 Page Not Found
- Search through the project for reference to `water_balance` or  the micro-site (there is still a link to it in the TR55 model page, but we'll get rid of that whole page soon anyway)

Connects #18

### CODE REVIEW

**Comment by rajadain** [comment]
The `bundle` script seems to be broken:

```bash
> kj bundle --debug
+ ARGS=--debug
+ vagrant ssh app -c 'cd /opt/app &&     envdir /etc/icp.d/env ./bundle.sh --debug'
./bundle.sh: line 192: syntax error: unexpected end of file
Connection to 127.0.0.1 closed.
kj: callProcess: /Users/ttuhinanshu/dev/bee/scripts/bundle.sh "--debug" (exit 2): failed
```

File: src/icp/bundle.sh [hunk 1]
```diff
@@ -186,8 +173,6 @@ VAGRANT_COMMAND="$TEST_COMMAND $VENDOR_COMMAND
     $NODE_SASS $ENTRY_SASS_FILE -o ${STATIC_CSS_DIR} &
     $BROWSERIFY $ENTRY_JS_FILES $BROWSERIFY_EXT $NUNJUCKS_TRANSFORM \
         -o ${STATIC_JS_DIR}main.js $EXTRA_ARGS &
```
Comment (rajadain) [inline src/icp/bundle.sh]: You need to remove the ampersand at the end here

**Comment by rajadain** [comment]
Had to do this to get the bundle script to work:

```diff
diff --git a/src/icp/bundle.sh b/src/icp/bundle.sh
index 8219ce3..998db36 100755
--- a/src/icp/bundle.sh
+++ b/src/icp/bundle.sh
@@ -159,7 +159,7 @@ if [ -n "$BUILD_VENDOR_BUNDLE" ]; then
         $BROWSERIFY $BROWSERIFY_REQ \
             -o ${STATIC_JS_DIR}vendor.js $EXTRA_ARGS &
         $BROWSERIFY $BROWSERIFY_REQ $BROWSERIFY_TEST_REQ \
-            -o ${STATIC_JS_DIR}test.vendor.js $EXTRA_ARGS &
+            -o ${STATIC_JS_DIR}test.vendor.js $EXTRA_ARGS &"
 fi
 
 TEST_COMMAND=""
@@ -172,7 +172,7 @@ fi
 VAGRANT_COMMAND="$TEST_COMMAND $VENDOR_COMMAND
     $NODE_SASS $ENTRY_SASS_FILE -o ${STATIC_CSS_DIR} &
     $BROWSERIFY $ENTRY_JS_FILES $BROWSERIFY_EXT $NUNJUCKS_TRANSFORM \
-        -o ${STATIC_JS_DIR}main.js $EXTRA_ARGS &
+        -o ${STATIC_JS_DIR}main.js $EXTRA_ARGS"
 
 # Ensure static asset folders exist.
 mkdir -p \
diff --git a/src/icp/sass/main.scss b/src/icp/sass/main.scss
index 2d53d65..5ccb18d 100644
--- a/src/icp/sass/main.scss
+++ b/src/icp/sass/main.scss
@@ -68,5 +68,4 @@
   "pages/draw",
   "pages/model",
   "pages/compare",
-  "pages/projects",
-  "pages/water-balance";
+  "pages/projects";
```

**Comment by arottersman** [comment]
Oh, that was really silly of me. I've pushed your changes.

**Pull request merged**